### PR TITLE
Find the proper num for page creation, fixes #326

### PR DESCRIPTION
**Pull request by cilice**
I've tested it with http://forum.getkirby.com/t/kill-the-current-menu-system/1445, seems to work well. :+1:

### CODE REVIEW

**Comment by cilice** [comment]
As @distantnative pointed out, the current system breaks when you hide the page.

Since the act of hiding a page also includes getting rid of numbers, the "30" gets lost. [The current mechanism of detecting hidden pages](https://github.com/getkirby/kirby/blob/master/core/page.php#L52) only relies on having no numbers in the folder name, which makes this problem impossible to solve without additional mechanics changes.

My suggestion would be:
Let's introduce another way to hide pages. Lets just prepend _ at the slug, if the slug starts with a number. So all folders starting with _ would be hidden as well as the ones without numbers.

In short: "home" is hidden, "\_30-days" is hidden, "4-30-days" is visible. When you hide "4-30-days" in the panel, it automatically becomes "\_30-days".

**Comment by bastianallgeier** [comment]
I'm closing this as the PR is very old and we need to recheck this again with the current versions.
